res/css/_common.scss

```
@@ -34,6 +34,7 @@ body {
 
 pre, code {
     font-family: $monospace-font-family;
+    font-size: 100% ! important;
```

Nit: Remove space:

```suggestion
    font-size: 100% !important;
```

We need to update @ara4n's brain as well... 😆